compiler/rustc_codegen_cranelift/example/alloc_example.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(start, core_intrinsics, alloc_error_handler, box_syntax)]
+#![feature(start, core_intrinsics, alloc_error_handler)]
```

Comment:
Have you verified that these examples still work? They look simple enough but it would be nice to make sure

Reply:
I've tried and failed before to get cranelift/gcc working 🙁 Would a try build check that these build?

Reply:
probably not, try builds check just about nothing by default and we don't test these backends in CI right now anyways.

Reply:
I can fix up cg_clif on the next sync if necessary. I hope to get test integration for cg_clif working on rust's CI soon, which would also make it easier to check it locally, but for now I need to fixup tests every once in a while myself.